src/libraries/System.Reflection.Metadata/src/System/Reflection/Metadata/Ecma335/Encoding/ControlFlowBuilder.cs

```diff
@@ -230,6 +257,25 @@ public void AddFilterRegion(LabelHandle tryStart, LabelHandle tryEnd, LabelHandl
 
         internal int ExceptionHandlerCount => _lazyExceptionHandlers?.Count ?? 0;
 
+        internal int RemainingSwitchBranches { get; set; }
+
+        internal void AssertNotInSwitch()
```

Comment:
nit: perhaps "ValidateNotInSwitch" instead. "Assert" just sounds like a `Debug.Assert` to me

Reply:
Fixed. I also called it in `DefineLabel` and added an assert that it throws.